firestore/src/include/firebase/firestore/map_field_value.h

```diff
@@ -31,6 +31,13 @@ using MapFieldValue = std::unordered_map<std::string, FieldValue>;
 /** @brief A map of `FieldValue`s indexed by field paths. */
 using MapFieldPathValue = std::unordered_map<FieldPath, FieldValue>;
 
+// Reference the following types so that they are included in the CXX
+// module. A workaround for deserialization cross reference errors
+// WIN-971.
```

Comment:
For upstreamability, we should avoid referencing an internal ticket, but we could reference a bug on an open-source Swift repo.

Reply:
@tristanlabelle do you mean a bug on apple/swift?

Reply:
@hjyamauchi yeah; when this was in JIRA, we would reference them as SR\d+.

Reply:
Done